code/stylesheet.dm

```diff
@@ -98,6 +98,7 @@ h1.alert, h2.alert		{color: #000000;}
 .solcom					{color: #22228B;}
 .changeling				{color: #800080;}
 .vox					{color: #AA00AA;}
+.vanax					{color: #941C1C;}

```

Comment:
![The color](https://puu.sh/zUQuV/b145c67755.png) For reference. Might be a touch close to sec?